core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -76,6 +79,28 @@
    * <code>avro.compatibility.level</code>
    */
   public static final String COMPATIBILITY_CONFIG = "avro.compatibility.level";
+  public static final String KAFKASTORE_SECURITY_PROTOCOL_CONFIG =
```

Comment:
Can we leverage the client security config support in Kafka to simplify a bunch of this (see `ConfigDef.withClientSslSupport`/`ConfigDef.withClientSaslSupport`)? Not sure it's possible given the additional prefix, but it seems easier to maintain and less error prone if we can just get the configs directly from Kafka instead of copy/pasting here.